main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -445,7 +445,13 @@ object Main {
 
       cmd("lsp").text("  starts the LSP server and listens on the given port.")
         .children(
-          arg[Int]("port").action((port, c) => c.copy(command = Command.Lsp(port)))
+          arg[Int]("port").action((port, c) => c.copy(command = Command.VSCodeLsp(port)))
+            .required()
+        )
+
+      cmd("lsp-vscode").text("  starts the VSCode-LSP server and listens on the given port.")
+        .children(
+          arg[Int]("port").action((port, c) => c.copy(command = Command.VSCodeLsp(port)))
             .required()
```

Comment:
Allowing both commands for a period allows us to transition the Flix VSCode extension without breakage, i.e.:

Release updated Flix
Release updated Flix Extension
Repurpose old command